docs/framework/wcf/troubleshooting-the-getting-started-tutorial.md

```diff
@@ -87,7 +87,7 @@ To turn off security:
 - For the client, in the configuration file, update the **\<security>** element under the **\<binding>** element as follows:  
   
     ```xml
-    <binding name="WSHttpBinding_ICalculator" security mode="None" />
+    <binding name="WSHttpBinding_ICalculator" security="None" />
```

Comment:
```suggestion
    <binding name="WSHttpBinding_ICalculator">
       <security mode="None" />
    <binding />
```

See https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/wcf/wshttpbinding for reference.